app/Resources/AppBundle/assets/js/main.js

```diff
@@ -2,11 +2,11 @@ require('./MovingCards');
 
 // materialize-css element functions
 $(document).ready(function(){
+    $(".dropdown-button").dropdown();
```

Comment:
Choose a good choice. I'm also fine with ```materialize.js``` which contains all adoptions to get it running (with symfony specific elements like knp menu bundle).